docs/src/main/asciidoc/datasource.adoc

```diff
@@ -625,6 +625,13 @@ Example:: `jdbc:mysql://localhost:3306/test`
 
 You can find more information about this feature and others detailed in the https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-reference.html[official documentation].
 
+==== MySQL Limitations
+
+When compiling a Quarkus application to native-image, the MySQL support for JMX and OCI integrations are disabled as they are not compatible
```

Comment:
```suggestion
When compiling a Quarkus application to native-image, the MySQL support for JMX and Oracle Cloud Infrastructure (OCI) integrations are disabled as they are not compatible
```

Reply:
@Sanne I've only proposed to expand what `OCI` means as it was not clear to me (as opposed to `JMX`)

Reply:
Fair enough - but be aware the MySQL docs and source code only refer to this as "OCI" so we should consistently include the `(OCI)` part (like you did here) in other references as well as I suspect many users know `OCI` without knowing the full name :)